client/src/test/java/no/unit/nva/model/client/ResourceDtoBuilderTest.java

```diff
@@ -0,0 +1,22 @@
+package no.unit.nva.model.client;
+
+
+import no.unit.nva.model.external.ResourceDto;
+import org.junit.jupiter.api.Test;
+
+import java.net.URI;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+class ResourceDtoBuilderTest {
```

Comment:
create a test which fails when a field is null. This test will ensure that all fields are included in the builder. we have a test method for it in test-utils. `doesNotHaveEmptyValues()`
Example:
`assertThat(actual,doesNotHaveEmptyValues())`